### PR TITLE
Add efficient creation of multiple documents

### PR DESCRIPTION
**Pull request by andrykonchin**
Add `Document.import` method to efficiently create multiple documents

`BatchWriteItem` ([documentation]( http://docs.aws.amazon.com/sdkforruby/api/Aws/DynamoDB/Client.html#batch_write_item-instance_method)) is used so we make fewer HTTP calls

Validation and callbacks are skipped for simplicity. `UnprocessedItems` and `ValidationException` are not handled too.

Example:
```
addresses = Address.import([{city: 'Chicago'}, {city: 'New York'}])
```

### CODE REVIEW

File: lib/dynamoid/persistence.rb [hunk 1]
```diff
@@ -372,7 +387,7 @@ def dump_field(value, options)
     # @since 0.2.0
     def persist(conditions = nil)
       run_callbacks(:save) do
-        self.hash_key = SecureRandom.uuid if self.hash_key.nil? || self.hash_key.blank?
+        self.hash_key = SecureRandom.uuid if self.hash_key.blank?
```
Comment (pboling) [inline lib/dynamoid/persistence.rb]: Thanks for catching that!

File: spec/dynamoid/persistence_spec.rb [hunk 2]
```diff
@@ -722,4 +722,60 @@ def self.name; 'Doc'; end
...
+    it 'makes batch operation' do
+      expect(Dynamoid.adapter.client).to receive(:batch_write_item).exactly(1).times.and_call_original
+      Address.import([{city: 'Chicago'}, {city: 'New York'}])
+    end
+
+    it 'makes multiple batch operations if the limit (25 items) exceeded' do
+      expect(Dynamoid.adapter.client).to receive(:batch_write_item).exactly(2).times.and_call_original
+      Address.import(Array.new(26, {city: 'Chicago'}))
+    end
+  end
```
Comment (pboling) [inline spec/dynamoid/persistence_spec.rb]: Great specs

File: README.md [hunk 3]
```diff
@@ -339,6 +339,13 @@ address.city = 'Chicago'
 address.save
 ```
 
+There is an efficient and low-level way to create multiple documents
+(without validation and callbacks running):
+
+```ruby
+users = User.import([{name: 'Josh'}, {name: 'Nick'}])
+```
+
```
Comment (pboling) [inline README.md]: Very nice!

**Comment by latortuga** [comment]
Any chance of a release with this feature? Could really use it 😄 👍 💯

**Comment by pboling** [comment]
@andrykonchin Can you add this to the changelog, and specify if it is a breaking change.  Following [semver 2.0](http://semver.org/), what should we release next?

**Comment by andrykonchin** [comment]
@pboling done

It isn't a breaking change. Just new method was added

> Following semver 2.0, what should we release next?
Do you mean which version?

TBH I don't know. Formally we have break changes (`find` and `save` behavior). But they are so minor and could be even considered as fixing to correspond to `Rails`/`Mongoid` default behavior

**Comment by pboling** [comment]
We should release a minor point update then.  I will release.

**Pull request merged**